tools/releases/helm.sh

```diff
@@ -100,28 +100,31 @@ function release {
 
   git clone --single-branch --branch "${GH_PAGES_BRANCH}" "$GH_REPO_URL"
 
+  CHART_TAR=$(find ${CHARTS_PACKAGE_PATH}/*.tgz -type f)
+  CHART_FILE=$(tar -tf ${CHART_TAR} | grep 'Chart.yaml')
+  CHART_VERSION=$(tar -zxOf ${CHART_TAR} ${CHART_FILE} | yq .version)
+
+  pushd ${GH_REPO}
```

Comment:
The `cr` commands now need to be executed under the `charts` directory to interact with the chart git workspace.